src/schema/user.js

```diff
@@ -244,4 +330,98 @@ export const resolvers = {
       );
     },
   },
+  Mutation: {
+    async addUserToUserDataService(parent, args, context, info) {
+      try {
+        const smaugUserId = context?.smaug?.user?.uniqueId;
+        if (!smaugUserId) {
+          throw "Not authorized";
+        }
+        await context.datasources.getLoader("userDataCreateUser").load({
+          smaugUserId: smaugUserId,
+        });
+        return { success: true };
+      } catch (error) {
+        return { success: false };
+      }
+    },
+    async deleteUserFromUserDataService(parent, args, context, info) {
+      try {
+        const smaugUserId = context?.smaug?.user?.uniqueId;
+        if (!smaugUserId) {
+          throw "Not authorized";
+        }
+
+        await context.datasources.getLoader("userDataDeleteUser").load({
+          smaugUserId: smaugUserId,
+        });
+        return { success: true };
+      } catch (error) {
+        return { success: false };
+      }
+    },
+
+    async setFavoritePickUpBranch(parent, args, context, info) {
+      try {
+        const { favoritePickUpBranch } = args;
+        const smaugUserId = context?.smaug?.user?.uniqueId;
+        if (!smaugUserId) {
+          throw "Not authorized";
+        }
+        if (isCPRNumber(smaugUserId)) {
+          throw "User not found in CULR";
+        }
+        await context.datasources
+          .getLoader("userDataFavoritePickupBranch")
+          .load({
+            smaugUserId: smaugUserId,
+            favoritePickUpBranch: favoritePickUpBranch,
+          });
+        return { success: true };
+      } catch (error) {
+        return { success: false };
+      }
+    },
+    async clearFavoritePickUpBranch(parent, args, context, info) {
+      try {
+        const smaugUserId = context?.smaug?.user?.uniqueId;
+        if (!smaugUserId) {
+          throw "Not authorized";
+        }
+        if (isCPRNumber(smaugUserId)) {
+          throw "User not found in CULR";
+        }
+        await context.datasources
+          .getLoader("userDataFavoritePickupBranch")
+          .load({
+            smaugUserId: smaugUserId,
+            favoritePickUpBranch: null,
+          });
+        return { success: true };
+      } catch (error) {
+        return { success: false };
+      }
+    },
+
+    async addOrder(parent, args, context, info) {
+      try {
+        const { orderId } = args;
+
+        const smaugUserId = context?.smaug?.user?.uniqueId;
+        if (!smaugUserId) {
+          throw "Not authorized";
+        }
+        if (isCPRNumber(smaugUserId)) {
+          throw "User not found in CULR";
+        }
+        await context.datasources.getLoader("userDataAddOrder").load({
+          smaugUserId: smaugUserId,
+          orderId: orderId,
+        });
+        return { success: true };
+      } catch (error) {
+        return { success: false };
+      }
+    },
+  },
```

Comment:
mangler der ikke en deleterord .. eller clearorder eller sådan noget ?

Reply:
Jo, jeg var lidt i tvivl om hvornår delete order skal kaldes. For tænker at det skal automatiseres på en måde. Men nu har jeg tilføjet en removeOrder